created the meal end point

codeironside opened the pull request:
# hi  @AsombaChinonso  and @dannygolden   😎😎


# PROKLIC-POS || CREATE MEAL ENDPOINT


## Project Description 👔


- worked on creating a meal endpoint


## Feature Description

- worked on the `meal` 🍢file 🗃 to create a route for uploading meals offered

  
  ### Happy Reviewing 🏌🏻

dannygolden reviewed (changes requested):
Hello, I think this is a generic data, you need to get info from me before you build the end points, the data captured here isn't sufficient. Also, terminologies matters e.g you used the shop instead of property.

AsombaChinonso commented:
Dear Team,

I trust you are doing well and great.

This is to acknowledged receipt of your email. I will look through and get
back to you where I need your input.

Thank you.

On Fri, Apr 7, 2023, 23:50 oluwatobi ayoola jolaosho <
***@***.***> wrote:

> hi @AsombaChinonso <https://github.com/AsombaChinonso> and @dannygolden
> <https://github.com/dannygolden> 😎😎 PROKLIC-POS || CREATE MEAL ENDPOINT Project
> Description 👔
>
>    - worked on creating a meal endpoint
>
> Feature Description
>
>    -
>
>    worked on the meal 🍢file 🗃 to create a route for uploading meals
>    offered
>    Happy Reviewing 🏌🏻
>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/SPARKBYTE-INTEGRATED/PROKLIC-POS/pull/1
> Commit Summary
>
>    - 7146bca
>    <https://github.com/SPARKBYTE-INTEGRATED/PROKLIC-POS/pull/1/commits/7146bca33f4ed6b53ec40cd70da6340dadfd4558>
>    created the meal end poin
>
> File Changes
>
> (6 files
> <https://github.com/SPARKBYTE-INTEGRATED/PROKLIC-POS/pull/1/files>)
>
>    - *A* server/controller/meals.js
>    <https://github.com/SPARKBYTE-INTEGRATED/PROKLIC-POS/pull/1/files#diff-8942d00dbebf8296c43cb94f6d0008c1dcc6cac164f5b77b7f2e886b8356b1fb>
>    (56)
>    - *M* server/middleware/authmiddleware.js
>    <https://github.com/SPARKBYTE-INTEGRATED/PROKLIC-POS/pull/1/files#diff-782a318d9cac2a199d4837a6d9b9188a9f8e4267f2690008b2fadce567e2afeb>
>    (1)
>    - *A* server/model/meals.js
>    <https://github.com/SPARKBYTE-INTEGRATED/PROKLIC-POS/pull/1/files#diff-94a4ef12e02591649778803083bb337c1645540f4d8712507e4a1bdd9d5a0dcf>
>    (41)
>    - *A* server/routes/meal.js
>    <https://github.com/SPARKBYTE-INTEGRATED/PROKLIC-POS/pull/1/files#diff-b53731fda3b2dadc7d89f754626ccbc7f70ebef46e46d838113885c24e9adaaf>
>    (13)
>    - *M* server/server.js
>    <https://github.com/SPARKBYTE-INTEGRATED/PROKLIC-POS/pull/1/files#diff-8cf1ffe3788af127768748703e2f15dcfb3a05ffd20b4c2375223637e3260938>
>    (2)
>    - *A* serverlog/.6e25589c67ea3fb03a32a59bc87872c31e875314-audit.json
>    <https://github.com/SPARKBYTE-INTEGRATED/PROKLIC-POS/pull/1/files#diff-973a3802c6516e013adabae95e268a6809f5fe4545383a61b28140d5501c2fcd>
>    (15)
>
> Patch Links:
>
>    - https://github.com/SPARKBYTE-INTEGRATED/PROKLIC-POS/pull/1.patch
>    - https://github.com/SPARKBYTE-INTEGRATED/PROKLIC-POS/pull/1.diff
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SPARKBYTE-INTEGRATED/PROKLIC-POS/pull/1>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIWDNZPJ5HVPAZOUMX63VUTXACK4XANCNFSM6AAAAAAWW7X6TQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

codeironside commented:
# hi  @AsombaChinonso  and @dannygolden   😎😎


# PROKLIC-POS || employee login


## Project Description 👔


- made necessary adjustment


## Feature Description

- changed `employee` login to username and password
  
  ### Happy Reviewing 🏌🏻

codeironside commented:
# hi  @AsombaChinonso  and @dannygolden   😎😎


# PROKLIC-POS || employee login


## Project Description 👔
 - made an enterprise end point
 -  correcting the employee registration endpoint


- made necessary adjustment


## Feature Description

- changed `employee` login to username and password
- updated `employee` registration information
- created an `enterprise` endpoint
  
  ### Happy Reviewing 🏌🏻
